src/Authoring/WinRT.SourceGenerator/WinRTTypeWriter.cs

```diff
@@ -274,24 +284,25 @@ public MappedType(Func<ISymbol, (string, string, string, bool, bool)> multipleMa
         {

```

Comment:
where is Windows.Foundation.Metadata.GuidAttribute <--> System.Runtime.InteropServices.Guid handled?  i.e., how does an authored interface specify an explicit guid?

Reply:
We don't currently have support for explicitly specifying a GUID, will open a issue to track that.

Reply:
are there any other custom attribute mappings we need to capture in an issue?

Reply:
Based on looking [here](https://github.com/microsoft/CsWinRT/blob/86895f639886517484ecc8db06ed036d3460dc96/src/cswinrt/code_writers.h#L1291) and the list of attributes, the only other special one is GCPressure which I don't believe applies here.  There is also AllowMultiple, but that is part of authoring attributes support.

Reply:
sounds good